packages/firebase_storage/example/test_driver/firebase_storage.dart

```diff
@@ -16,6 +17,7 @@ void main() {
     FirebaseStorage firebaseStorage;
 
     setUp(() async {
+      await Firebase.initializeApp();
       firebaseStorage = FirebaseStorage();
```

Comment:
If storage is migrated to the new core, would this still be the way to get a storage instance? I would expect it to be something like `firebaseStorage = FirebaseStorage.instance`?

Reply:
New core is `await Firebase.initializeApp();`, the instance getter for each plugin is being updated when we rework the plugin itself (to handle `.instance` and `instanceFor()`).

Reply:
OK cool, so the move to the new instance getter will be in a follow up PR?

Reply:
Yep, once we move to Storage it'll be handled there.